docs/source/overview/getting_started.rst

```diff
@@ -76,6 +76,8 @@ Standalone JSON/REST Server
 
 The standalone server is a lightweight way to try out AtlasDB and can be
 done by running AtlasDbServiceServer in the atlasdb-service-server project.
+The default configuration file uses an ephemeral in-memory DB;
+if you'd like to run tests with persisted data there is also an `atlasdb_standalone_postgres.yml`
```

Comment:
Do we think this is visible enough? I'd ideally give it its own attention box (probably at info), though it might well be the case that no one uses this